docker/README.md

```diff
@@ -0,0 +1,130 @@
+# 🪐 Docker Image and Container for Validate Tool
+
+The docker image of Validate Tool provides an easy way to deploy the Validate Tool, without having to 
+worry about the Java/JDK versions installed and operating systems supported. Also, this helps to 
+easily use the Validate Tool in other execution environments such as docker compose and Amazon ECS.
+
+The only prerequisite to use the dockerized Validate Tool is the availability of docker software in the host computer.
+At the time of writing this, the Validate Tool was tested with Docker version 20.10.12.
+
+## 🏃 Steps to build the docker image of the Validate Tool
+
+1. Clone the GIT Repository containing the validate tool.
+```shell
+git clone https://github.com/NASA-PDS/validate.git
+```
+
+2. Update (if required) the following version in the `Dockerfile` with the required Validate Tool version.
+
+| Variable            | Description |
+| ------------------- | ------------|
+| validate_version     | The version of the Validate Tool release to be included in the docker image |
+
+```    
+# Set following argument with the required validate tool version
+ARG validate_version=3.0.3
+```
+
+3. Open a terminal and change the current working directory to `validate/docker`.
+
+4. Build the docker image as follows.
+
+```
+    docker image build -t nasapds/validate .
```

Comment:
In documentation, I really like to see the full arguments like `--tag` instead of `-t` 😇